index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="header">
+        <a href="#Home" class="active"> My Logo</a>
+        <a href=" " class="icon">
+            <i class="fa-solid fa-bars fa-2x"></i>
+        </a>
+    </div>
+    <div class="headline-section">
+        <h1 class="txt"> I'm Dev Mike <br>
+            Glad to see<br>you!</h1>
+        <p class="introduction"> I'm a software developer I can help you build a project a product,feature or
+            website Look through some
+            of my work and <br> experience!If you like what you see and have a project you need coded don't hestiate
+            to contact me</p>
+        <div class="socials">
+            <h2>LET'S CONNECT</h2>
+            <div class="icons">
+                <a href=""> <i class="fa-brands fa-twitter fa-1x" background: #505F79></i></a>
+                <a href=""> <i class="fa-brands  fa-linkedin fa-1x "></i></a>
+                <a href=""> <i class="fa-brands fa-github fa-1x"></i></a>
+                <a href=""> <i class="fa-brands fa-m fa-1x"></i></a>
+                <a href=""><i class="fa-solid fa-hand-peace fa-1x"></i></a>
+            </div>
+        </div>
+    </div>
```

Comment:
- For semantic reasons, I suggest you wrap this section in a `section` tag. It represents a generic standalone section of the document, so consider using it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.
- There are some little differences in design between your Figma template and your portfolio design. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Figma design:
- Please take note of the marked section from the image attached below
![rjrjrj](https://user-images.githubusercontent.com/73607512/205687626-b3ff2468-dccd-479c-9fee-0da7a3276bbd.PNG)

Your design:
![yrkryjry](https://user-images.githubusercontent.com/73607512/205687770-dad43acd-55ad-47c9-9347-31a25b497733.PNG)

Reply:
@mckent05 every changes commited